pvacseq/lib/main.py

```diff
@@ -45,6 +45,15 @@ def main(args_input = sys.argv[1:]):
                         default=21)
     parser.add_argument('-g', '--gene-expn-file', help='genes.fpkm_tracking file from Cufflinks')
     parser.add_argument('-i', '--transcript-expn-file', help='isoforms.fpkm_tracking file from Cufflinks')
+    parser.add_argument('--net-chop-method',
+                        choices=lib.net_chop.methods,
+                        help="NetChop prediction method to use (\"cterm\" for C term 3.0, \"20s\" for 20S 3.0).  Default: \"cterm\" (C term 3.0)",
+                        default=None)
+    parser.add_argument(
```

Comment:
Is it intentional that the alignment/formatting here doesn't match the other arguments?